kbase-extension/static/kbase/js/widgets/appWidgets2/validation.js

```diff
@@ -630,6 +639,29 @@ define([
             return value;
         }
 
+        function importIntString(value, nonIntError) {
```

Comment:
This is mainly used for initializing the int input widget. Effectively unchanged from `validators/int`